modules/core/src/v2/wallet.ts

```diff
@@ -1257,6 +1259,8 @@ export class Wallet {
    * @param {Number} params.count=1 number of new addresses which should be created (maximum 250)
    * @param {Number} params.forwarderVersion The version of address to create, if applicable
    * @param {Boolean} params.lowPriority Ethereum-specific param to create address using low priority fee address
+   * @param {String} params.baseAddress base address of the wallet(optional parameter)
+   * @param {Boolean} params.strictMode set to strictly verify the created address
```

Comment:
please explain the meaning of this in more detail (the condition in which we skip the verification)

Reply:
Done with the new commit.